### ISSUE TITLE
Installed in Docker

### ISSUE THREAD

**Issue by PauFCB** [LMS-Community/plugin-LocalPlayer#2]
Hi, I have LMS installed on Docker and want to use MacBook to listen to music. How can I configure this plugin to do this? Thanks

**Comment by terual** [comment]
I do not think that LMS in Docker will have access to your sound device. Maye it is easier to just run squeezelite on your MacBook for playback.
